app/controllers/account_details.js

```diff
@@ -85,11 +86,7 @@ export default Ember.Controller.extend({
         }
       });
     } else if (onlineOrder) {
-      this.transitionToRoute("request_purpose", {
-        queryParams: {
-          bookAppointment: false
-        }
-      });
+      this.transitionToRoute("submitted_orders");
```

Comment:
Just a suggestion, maybe renaming the page to something clearer, like `submitted_order_selection`

`submitted_orders` could imply that it's just a list (similar to the dashboard), but it's actually more a form

Reply:
Yeah... changed its name to `sumitted_order_selection`.